amc2moodle.sh

```diff
@@ -88,17 +88,17 @@ EOF
 # function to init the requiered variable
 function init() {
     # get absolute name of the input file
-    filein=$(readlink -m $OPTARG) # do not check existance!
+    filein=$(realpath $OPTARG) # do not check existance!
```

Comment:
Could you also replace the other readlink

Reply:
done but not necessary (see new proposition with Python script)

Reply:
done but not necessary from now (see my new script proposition)